common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -319,9 +346,16 @@ def get_pending_reviews(self, user_id):
         url = self.pending_reviews_url
         student_uid = self.generate_student_uid(user_id)
         payload = {'student_uid': student_uid, 'nested': True}
-        response = requests.get(url, headers=self.request_headers, data=payload)
-        if response.content == 'No Student Found':
+        try:
+            response = requests.get(url, headers=self.request_headers, data=payload)
+            content = response.content
```

Comment:
@macornwell As I mentioned [here](https://github.com/open-craft/edx-platform/pull/69#discussion_r148738976), we should remove this line and stick to referencing `response.content` directly in the `else` portion of the try-except-else block.